vehicles/src/common/helper/permit-fee.helper.ts

```diff
@@ -99,6 +97,24 @@ export const yearlyPermit = (duration: number): boolean => {
   return duration <= 365 && duration >= 361;
 };
 
+export const calculateDuration = (application: Permit): number => {
+  let startDate = application.permitData.startDate;
+  const endDate = application.permitData.expiryDate;
+  const today = dayjs(new Date()).format('YYYY-MM-DD');
+  if (
+    application.permitStatus === ApplicationStatus.VOIDED &&
+    today < startDate
+  )
+    startDate = today;
```

Comment:
I don't really understand this logic... Why do we need to assign today to startDate when today is before start date? Shouldn't it take the later of the two to derive how many days are left for the permit?

Reply:
I meant startDate < today, mistake on my part. correcting